<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserLoginDialog</class>
 <widget class="QDialog" name="UserLoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="backgroundLbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>712</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../resources/images/images.qrc">:/images/screens/form_login.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="loginBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>589</y>
     <width>135</width>
     <height>78</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resources/images/images.qrc">
     <normaloff>:/images/screens/button_login.png</normaloff>:/images/screens/button_login.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>135</width>
     <height>78</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameLE">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>493</y>
     <width>175</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLE">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>534</y>
     <width>175</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../resources/images/images.qrc"/>
 </resources>
 <connections/>
</ui>
